disco_aws_automation/disco_autoscale.py

```diff
@@ -207,7 +207,7 @@ def create_autoscale_tags(group_name, tags):
         return [boto.ec2.autoscale.Tag(key=key, value=value, resource_id=group_name, propagate_at_launch=True)
                 for key, value in tags.iteritems()] if tags else None
 
-    def update_group(self, group, launch_config, vpc_zone_id=None,
+    def modify_group(self, group, launch_config, vpc_zone_id=None,
```

Comment:
Function name changed because I had to reuse the name for the new method with the same name for both Autoscale and Elastigroup.